packages/hyperion-autologging/src/ALSurface.ts

```diff
@@ -248,14 +248,14 @@ export function init(options: InitOptions): ALSurfaceHOC {
       }
     } else {
       fullSurfaceString = proxiedContext.surface;
-      fullRegionString = proxiedContext.region;
+      fullRegionString = proxiedContext.nonInteractiveSurface;
       domAttributeName = domSurfaceAttributeName
       domAttributeValue = fullSurfaceString;
     }
 
     const surfaceData: SurfaceData = {
       surface: fullSurfaceString,
-      region: fullRegionString,
+      nonInteractiveSurface: fullRegionString,
```

Comment:
If renaming, can we fully replace "region" with "nonInteractiveSurface"?

Reply:
Yes, I'll do that. Thanks for catching it. I relied on IDE doing all the renames.